tests/functional/lib/helpers.js

```diff
@@ -804,8 +838,6 @@ define([
             })
           .end()
 
-          .then(takeScreenshot())
```

Comment:
`testElementExists` now takes a screenshot, this would have caused 2 to be taken.